app/test/spec/zeebe-api-spec.js

```diff
@@ -682,6 +682,354 @@ describe('ZeebeAPI', function() {
   });
 
 
+  describe('#getTopology', function() {
+
+    it('should set success=true if topology was retrieved', async () => {
+
+      // given
+      const zeebeAPI = mockZeebeNode();
+
+      const parameters = {
+        endpoint: {
+          type: 'selfHosted',
+          url: 'https://google.com'
+        }
+      };
+
+      // when
+      const result = await zeebeAPI.checkConnection(parameters);
+
+      // then
+      expect(result.success).to.be.true;
+    });
+
+
+    it('should return TopologyResponse if topology was retrieved', async () => {
+
+      // given
+      const topologyResponse = { clusterSize: 3, gatewayVersion: '0.26.0' };
+
+      const zeebeAPI = mockZeebeNode({
+        ZBClient: function() {
+          return {
+            topology: function() {
+              return topologyResponse;
+            }
+          };
+        }
+      });
+
+      const parameters = {
+        endpoint: {
+          type: 'selfHosted',
+          url: 'https://google.com'
+        }
+      };
+
+      // when
+      const result = await zeebeAPI.getTopology(parameters);
+
+      // then
+      expect(result.success).to.be.true;
+      expect(result.response).to.equal(topologyResponse);
+    });
+
+
+    it('should set success=false on failure', async () => {
+
+      // given
+      const zeebeAPI = mockZeebeNode({
+        ZBClient: function() {
+          return {
+            topology: function() {
+              throw new Error('TEST ERROR.');
+            }
+          };
+        }
+      });
+
+      const parameters = {
+        endpoint: {
+          type: 'selfHosted',
+          url: 'https://google.com'
+        }
+      };
+
+      // when
+      const result = await zeebeAPI.getTopology(parameters);
+
+      // then
+      expect(result.success).to.be.false;
+      expect(result.response).to.be.undefined;
+    });
+
+
+    describe('should return correct error reason on failure', function() {
+
+      it('for <endpoint-unavailable>', async () => {
+
+        // given
+        const zeebeAPI = mockZeebeNode({
+          ZBClient: function() {
+            return {
+              topology: function() {
+                throw new NetworkError('TEST ERROR.', 14);
+              }
+            };
+          }
+        });
+
+        const parameters = {
+          endpoint: {
+            type: 'selfHosted'
+          }
+        };
+
+        // when
+        const result = await zeebeAPI.checkConnection(parameters);
```

Comment:
Why do we call `checkConnection` and not `getTopology` in this describe block?

Reply:
Thanks! This was by mistake